### PR TITLE
Rework Website

### PR DESCRIPTION
**Pull request by adius**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Brickify",
   "version": "0.9.2",
-  "description": "Extend your LEGO with 3D-printed parts!",
+  "description": "Faster 3D-printing with LEGO bricks",
```
Comment (AtiX) [inline package.json]: LEGO(r)-bricks

File: views/rootLayout.jade [hunk 2]
```diff
@@ -2,15 +2,10 @@ doctype html
 html
 	head
 		meta(charset='utf-8')
-		if pageTitle != null
-			title #{pageTitle} - brickify - Extend your LEGO with 3D-printed parts!
-		else
-			title brickify - Extend your LEGO with 3D-printed parts!
+		title #{pageTitle ? pageTitle + ' - ' : ''}Brickify
```
Comment (AtiX) [inline views/rootLayout.jade]: Slogan is missing - needs to be there for SEO

File: views/landingpage/footer.jade [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-		|LEGO#[sup &reg;] is a trademark of the LEGO Group of companies which
-		|does not sponsor, authorize or endorse this site
-		br
-		|This product uses the Thingiverse API but is not endorsed or certified by
-		|Thingiverse or its affiliates.
+	
+	p#legal.
+		LEGO#[sup &reg;] is a trademark of the LEGO Group of companies which
+		does not sponsor, authorize or endorse this site.
+		<br>
```
Comment (Steditor) [inline views/landingpage/footer.jade]: No HTML in jade pls

**Comment by Steditor** [comment]
:boat:

**Pull request merged**